packages/vega/package.json

```diff
@@ -43,33 +43,33 @@
     "postpublish": "./schema-deploy.sh"
   },
   "dependencies": {
-    "vega-crossfilter": "~5.0.0",
-    "vega-dataflow": "~6.0.0",
-    "vega-encode": "~5.0.0",
+    "vega-crossfilter": "~5.1.0",
+    "vega-dataflow": "~6.1.0",
+    "vega-encode": "~5.1.0",
     "vega-event-selector": "~4.0.0",
-    "vega-expression": "~6.0.0",
-    "vega-force": "~5.0.0",
+    "vega-expression": "~6.1.0",
+    "vega-force": "~5.1.0",
     "vega-format": "~2.0.0",
-    "vega-functions": "~6.0.0",
-    "vega-geo": "~5.0.0",
-    "vega-hierarchy": "~5.0.0",
-    "vega-label": "~2.0.0",
-    "vega-loader": "~5.0.0",
-    "vega-parser": "~7.0.0",
+    "vega-functions": "~6.1.0",
+    "vega-geo": "~5.1.0",
+    "vega-hierarchy": "~5.1.0",
+    "vega-label": "~2.1.0",
+    "vega-loader": "~5.1.0",
+    "vega-parser": "~7.1.0",
     "vega-projection": "~2.0.0",
-    "vega-regression": "~2.0.0",
-    "vega-runtime": "~7.0.0",
+    "vega-regression": "~2.1.0",
+    "vega-runtime": "~7.1.0",
     "vega-scale": "~8.0.0",
-    "vega-scenegraph": "~5.0.0",
+    "vega-scenegraph": "~5.1.0",
     "vega-statistics": "~2.0.0",
     "vega-time": "~3.0.0",
-    "vega-transforms": "~5.0.0",
-    "vega-typings": "~2.0.0",
+    "vega-transforms": "~5.1.0",
+    "vega-typings": "~2.1.0",
     "vega-util": "~2.0.0",
```

Comment:
I think we should also bump the version of `vega-util` to `2.1.0` , since this release adds a new utility to it.

Reply:
+1 
@lsh 
would you bump up the version for vega-util to include https://github.com/vega/vega/pull/4127?